inst/odin/lancelot.R

```diff
@@ -1746,16 +1850,36 @@ vaccine_probability_doses[, ] <- min(
   as.numeric(1))
 dim(vaccine_probability_doses) <- c(n_groups, n_doses)
 
-vaccine_attempted_doses[, ] <- vaccine_missed_doses[i, j] + (
-  if (as.integer(step) >= dim(vaccine_dose_step, 3) ||
-      vaccine_n_candidates[i, j] == 0) 0
+tmp_attempted_doses[, ] <- vaccine_missed_doses[i, j] + (
```

Comment:
I'd suggest renaming tmp_attempted_doses to total_attempted_doses
then clarify that you split the total into the "direct" and the "skip" parts

Reply:
I've changed this, and added comments